### PR TITLE
docs: upgrade_0.8.0

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by lqy222** [comment]
Below feature not be tested:
 - Support the new built-in functions `var_samp`, `var_pop`, `entropy`, `earth_distance`, `nth_value_where` and `add_months` (#3046 #3193 @aceforeverd)
- ~~Support register table in Spark and OpenMLDB (#3057 @tobegit3hub)~~
- ~~Support batch read for openmldb-spark-connector (#3070 @tobegit3hub)~~
- Support kubernetes as Taskmanager backend (#3147 #3157 #3185 @tobegit3hub)
- Support `s3` for offline engine (#3229 @tobegit3hub)
- Enhance the diagnostic tool (#3224 #3208 @zhangziheng01233)
- ~~Add `SKIP_INDEX_CHECK` option to skip index checking when deploying SQLs (#3241 @dl239)~~
- Support symbolic paths for offline table (#3235 @tobegit3hub)

bugfix
- ~~There is curator conflict problem when executing offline SQLs under certain circumstances. (#3090 @tobegit3hub)~~
- ~~`SELECT ... INTO ...` statement execution fails in offline mode if there is no source table. (#3116 @tobegit3hub)~~

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## [0.8.0] - 2023-05-10
+
+### Features
+- Add a new synchronization tool to automatically synchronize data from online storage to offline storage (#3256 @vagetablechic)
+- Support the new built-in functions `var_samp`, `var_pop`, `entropy`, `earth_distance`, `nth_value_where` and `add_months` (#3046 #3193 @aceforeverd)
+- Support batch read for openmldb-spark-connector (#3070 @tobegit3hub)
+- Support Kubernetes as a TaskManager backend for the offline engine (#3147 #3157 #3185 @tobegit3hub)
+- Support getting yarn log for `SHOW JOBLOG` (#3163 @tobegit3hub)
```
Comment (lumianph) [inline CHANGELOG.md]: this should be removed once the PR reverted

Comment (dl239) [reply CHANGELOG.md]: ok

**Review by lumianph — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**